nmdc_runtime/site/ops.py

```diff
@@ -527,29 +528,43 @@ def perform_mongo_updates(context, json_in):
     if rv["result"] == "errors":
         raise Failure(str(rv["detail"]))
 
-    coll_has_id_index = collection_indexed_on_id(mongo.db)
-    if all(coll_has_id_index[coll] for coll in docs.keys()):
+    add_docs_result = _add_schema_docs_with_or_without_replacement()
```

Comment:
Hi @dwinston and @PeopleMakeCulture, I think this line is the problematic one, with respect to the issue @pkalita-lbl reported on [Slack](https://nmdc-group.slack.com/archives/C0739GREU9G/p1723222294109459). On this line, the function is being invoked without arguments, but the function's signature lists to required parameters.

Reply:
Based upon the various variable names around here, I suspect the author of this line meant to write:

```suggestion
    add_docs_result = _add_schema_docs_with_or_without_replacement(mongo, docs)
```